apollo-federation/src/sources/connect/expand/tests/snapshots/it_expand_supergraph@keys.graphql-2.snap

```diff
@@ -501,19 +501,21 @@ input_file: apollo-federation/src/sources/connect/expand/tests/schemas/expand/ke
                         None,
                     ),
                     Path(
-                        Alias {
-                            name: WithRange {
-                                node: Field(
-                                    "id2",
-                                ),
+                        Some(
+                            Alias {
+                                name: WithRange {
+                                    node: Field(
+                                        "id2",
+                                    ),
+                                    range: Some(
+                                        3..6,
+                                    ),
+                                },
                                 range: Some(
-                                    3..6,
+                                    3..7,
```

Comment:
Why did the location of these change?

Reply:
Good question! My heart skipped a beat when I saw this, but it's a harmless diff artifact.

In this example, the output before looked like
```
                    Path(
                        Alias {
                            name: WithRange {
                                node: Field(
                                    "id2",
                                ),
                                range: Some(
                                    3..6,
                                ),
                            },
                            range: Some(
                                3..7,
                            ),
                        },
                        PathSelection {
                          # omitting this for brevity
                        },
                    ),
```

Since we changed the `NamedSelection::Path` enum to have an `Option<Alias>` as its first argument, there's a new layer of wrapping, so the new output looks like this:
```
                    Path(
                        Some(
                            Alias {
                                name: WithRange {
                                    node: Field(
                                        "id2",
                                    ),
                                    range: Some(
                                        3..6,
                                    ),
                                },
                                range: Some(
                                    3..7,
                                ),
                            },
                        ),
                        PathSelection {
                          # omitting this for brevity
                        },
                    ),
```

All that's really happening here is the `Alias` from before is getting wrapped with `Some(...)`, and the ranges are staying the same.

If you drop these two snippets into different files and diff them, however, the diff logic somehow decides the inner range from the first corresponds to the outer range from the second, so the `3..6` appears to be changing in place (to `3..7`):
```diff
diff --git a/before.snap b/after.snap
index 6d12da5fe..9142140d8 100644
--- a/before.snap
+++ b/after.snap
@@ -1,17 +1,19 @@
                     Path(
-                        Alias {
-                            name: WithRange {
-                                node: Field(
-                                    "id2",
-                                ),
+                        Some(
+                            Alias {
+                                name: WithRange {
+                                    node: Field(
+                                        "id2",
+                                    ),
+                                    range: Some(
+                                        3..6,
+                                    ),
+                                },
                                 range: Some(
-                                    3..6,
+                                    3..7,
                                 ),
                             },
-                            range: Some(
-                                3..7,
-                            ),
-                        },
+                        ),
                         PathSelection {
                           # omitting this for brevity
                         },
```

If you run the diff with `-w` to ignore whitespace, the amount of change more closely matches what's logically happening:
```diff
diff --git a/before.snap b/after.snap
index 6d12da5fe..9142140d8 100644
--- a/before.snap
+++ b/after.snap
@@ -1,18 +1,20 @@
                     Path(
+                        Some(
                             Alias {
                                 name: WithRange {
                                     node: Field(
                                         "id2",
                                     ),
                                     range: Some(
                                         3..6,
                                     ),
                                 },
                                 range: Some(
                                     3..7,
                                 ),
                             },
+                        ),
                         PathSelection {
                           # omitting this for brevity
                         },
                     ),
```